HW1/First lesson.playground/Contents.swift

```diff
@@ -0,0 +1,217 @@
+import UIKit
+
+//first task
+protocol HomeworkService {
+    // Функция деления с остатком, должна вернуть в первой части результат деления, во второй части остаток.
+    func divideWithRemainder(_ x: Int, by y: Int) -> (Int, Int)
+
+    // Функция должна вернуть числа фибоначчи.
+    func fibonacci(n: Int) -> [Int]
+
+    // Функция должна выполнить сортировку пузырьком.
+    func sort(rawArray: [Int]) -> [Int]
+
+    // Функция должна преобразовать массив строк в массив первых символов строки.
+    func firstLetter(strings: [String]) -> [Character]
+
+    // Функция должна отфильтровать массив по условию, которое приходит в параметре `condition`. (Нельзя юзать `filter` у `Array`)
+    func filter(array: [Int], condition: ((Int) -> Bool)) -> [Int]
+}
+
+struct Service: HomeworkService{
+    func divideWithRemainder(_ x: Int, by y: Int) -> (Int, Int) {
+        let first = x/y
+        let second = x%y
+        return(first, second)
+    }
+    
+    func fibonacci(n: Int) -> [Int] {
+        var result = [0, 1]
+        
+        guard n > 1 else {
```

Comment:
желательно в одну строчки 
```suggestion
        guard n > 1 else { return }
```